src/agent/src/config.rs

```diff
@@ -198,6 +219,12 @@ impl Default for AgentConfig {
             guest_components_procs: GuestComponentsProcs::default(),
             #[cfg(feature = "guest-pull")]
             image_registry_auth: String::from(""),
+            #[cfg(feature = "guest-pull")]
+            enable_signature_verification: false,
```

Comment:
Something to be aware of - in `CCv0` we decided for security sack to default to have verification on. You've changed this, which I think is probably the right decision due to the impact it will have on vanilla kata containers, but we want to ensure that we strongly document that somewhere in CoCo that it is opt-in.

Reply:
Yes. I made an issue on CoCo side https://github.com/confidential-containers/confidential-containers/issues/221